v8/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/elliotchance/redismock
```

Comment:
Shouldn't this be `module github.com/elliotchance/redismock/v8`?

Reply:
You're right! I fixed it.